src/routes/chunk/handlers.test.ts

```
@@ -0,0 +1,238 @@
+      }),
+    );
+
+    await request(app)
+      .get('/chunk/1234')
+      .expect(404)
+      .then((res: any) => {
+        assert.strictEqual(res.status, 404);
+      });
+  });
+
+  it.only('should return 500 if chunk data source returns undefined', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)